clang/test/Sema/return-mismatch.c

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -Wno-return-type -Wreturn-mismatch -fsyntax-only -verify %s
+
+// Test that -Wreturn-mismatch is enabled and -Wreturn-type is disabled.
+
+int foo(void) __attribute__((noreturn));
+int bar(void);
+
+void test1() {
```

Comment:
```suggestion
void test1(void) {
```
Same change should be made to all the functions in this file.

Reply:
done